Userland/Libraries/LibWeb/HTML/Scripting/ClassicScript.cpp

```diff
@@ -56,7 +56,7 @@ JS::NonnullGCPtr<ClassicScript> ClassicScript::create(ByteString filename, Strin
     // 11. If result is a list of errors, then:
     if (result.is_error()) {
         auto& parse_error = result.error().first();
-        dbgln_if(HTML_SCRIPT_DEBUG, "ClassicScript: Failed to parse: {}", parse_error.to_byte_string());
+        dbgln_if(HTML_SCRIPT_DEBUG, "ClassicScript: Failed to parse: {}", parse_error.to_string());
```

Comment:
`to_string` returns `ErrorOr` which affects the formatting I think?

Reply:
Now it doesn't 😸 !